tests/integration/reqs/test_subscribe.py

```diff
@@ -1,17 +1,24 @@
 from tests.integration.integration_test_case import IntegrationTestCase
-from tests.integration.it_utils import test_async_and_sync
+from tests.integration.it_utils import (
+    accept_ledger_async,
+    sign_and_reliable_submission_async,
+    test_async_and_sync,
+)
+from tests.integration.reusable_values import DESTINATION, WALLET
 from xrpl.models.requests import StreamParameter, Subscribe
 from xrpl.models.requests.unsubscribe import Unsubscribe
+from xrpl.models.transactions.payment import Payment
 
 _MESSAGE_LIMIT = 3
 
 
 class TestSubscribe(IntegrationTestCase):
-    @test_async_and_sync(globals(), websockets_only=True, use_testnet=True)
+    @test_async_and_sync(globals(), websockets_only=True)
     async def test_ledger_subscription(self, client):
         await client.send(Subscribe(streams=[StreamParameter.LEDGER]))
         count = 0
         async for message in client:
```

Comment:
Might be worth a comment explaining that a default message comes from the client - otherwise someone reading the test later on might get confused and think that there's an issue with this line of these tests